uniffi_bindgen/src/interface/types/mod.rs

```diff
@@ -146,10 +152,20 @@ impl From<&Type> for FFIType {
             | Type::Map(_)
             | Type::Timestamp
             | Type::Duration => FFIType::RustBuffer,
+            Type::External { .. } => FFIType::RustBuffer,
         }
     }
 }
 
+// The "kind" of external type.
+#[derive(Debug, Clone, Eq, PartialEq, Hash, Ord, PartialOrd)]
+pub enum ExternalTypeKind {
+    // A type defined by UniFFI from a different crate that this crate `use`s
+    Uniffi { crate_name: String },
+    // A manually hand-written type that wraps the specified primitive.
+    Wrapping { primitive: Box<Type> },
```

Comment:
I have an implementation for this in mind -- it just needs 1 more attribute: `ffi_converter_name: String`.

I also have a dream to extend this to: 
```
  Wrapping {
    ffi_converer_name: String,
    primitive: Box<Type>,
    python_template_string: Option<String>,
    python_create_ffi_converter: Option<String>,
    kotlin_template_string: Option<String>,
    kotlin_ffi_converter_name: Option<String>,
    # repeat for other languages
}
```

Reply:
That makes sense! Although the language specific stuff looks like maybe it should be inside the various [config structs](https://github.com/mozilla/uniffi-rs/blob/main/uniffi_bindgen/src/bindings/mod.rs#L68) and maybe specified [in the toml](https://github.com/mhammond/uniffi-rs/commit/40be445c85cbcb31631a7d0f4c539eba63860ee0#diff-eb0847bbe6a9781ee040e3ef30d46ec9fa8bc5a547ff56ca8fcf58f6c365decd)?